examples/src/read-data.c

```diff
@@ -85,15 +85,15 @@ static void alloc_buf(unsigned long length)
 
 static void do_pread(int fd, size_t length, off_t offset)
 {
-    int ret = 0;
+    ssize_t ret = 0;
 
     alloc_buf(length);
 
     errno = 0;
 
     ret = pread(fd, buf, length, offset);
 
-    printf(" -> pread(off=%lu, len=%lu) = %d", offset, length, ret);
+    printf(" -> pread(off=%lu, len=%lu) = %ld", offset, length, ret);
```

Comment:
ssize_t uses '%zd' for format string, size_t uses '%zu'

Reply:
Thanks, @MichaelBrim. I've updated.